x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/IndexLifecycleExplainResponse.java

```diff
@@ -412,11 +415,11 @@ public Integer getFailedStepRetryCount() {
         return failedStepRetryCount;
     }
 
-    public TimeValue getAge() {
+    public TimeValue getAge(Supplier<Long> now) {
```

Comment:
Just as an aside, I think it would be best practice to use `LongSupplier` here. Since this is a backport, I think it's best to stick with what you have — it obviously doesn't deserve either introducing a diff between `8.x` and `main`, or doing an extra PR on `main`. So just consider this me flying the flag for this under-loved class of interfaces!